modules/oadp-support-backup-data-immutability.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/installing/about-installing-oadp.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-support-backup-data-immutability_{context}"]
+= OADP does not support backup data immutability
+
+Starting with {oadp-short} 1.3, backups may not function as expected when the target object storage has an immutability option configured. These immutability options are referred to by different names, for example:
+
+* object lock
+* object retention
+* bucket versioning
+* Write Once Read Many (WORM) buckets
+
+The primary reason for the absence of support lies in the fact that {oadp-short} initially saves the state of a backup as _finalizing_ and then scrutinizes whether any async operations are in progress. If there are async operations in progress, {oadp-short} needs to wait for all of them to be finished before moving the backup state to _complete_. If there are no async operations, and the state is moved to _complete_ immediately. Regardless, {oadp-short} must modify the metadata in object storage, which will not be possible if some kind of immutability is configured on the object storage.
+
+With versions before {oadp-short} 1.3, there was no explicit support in {oadp-short} to work with object storage with an 'immutability' configuration. You may see some problems even though backups seem to work, for example, version objects not being deleted when backup is deleted.
+
+[NOTE]
+====
+Depending on the specific provider and configuration, backups may still work in some cases.
+====
+
+* AWS S3 service supports backups because S3's object lock only applies to versioned buckets, and the object data can still be updated as the new version. However, when backups are deleted, old versions of the objects will not be deleted.
```

Comment:
```suggestion
* AWS S3 service supports backups because S3 object lock only applies to versioned buckets, and you can still update the object data for the new version. However, when backups are deleted, old versions of the objects are not deleted.
```